v2/tools/asoctl/pkg/crd/crd_clean.go

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package crd
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+
+	"github.com/Azure/azure-service-operator/v2/tools/asoctl/internal/client"
+	"github.com/Azure/azure-service-operator/v2/tools/asoctl/internal/crd"
+)
+
+func newCRDCleanCommand() (*cobra.Command, error) { //nolint:unparam
```

Comment:
Actually, should we just remove the `error` return here? I'm not seeing the case where the _creation of the cmd_ will fail. Execution yes of course (that's `RunE` below), but constructing the `cmd` object seems pretty unlikely to hit an error? Then we could drop the `unapram` ignoring here.